### PR TITLE
MQTT streaming: Using public PropsAdapter API

### PR DESCRIPTION
**Pull request by jcroig**
## References

References #2088

### CODE REVIEW

File: mqtt-streaming/src/main/scala/akka/stream/alpakka/mqtt/streaming/scaladsl/MqttSession.scala [hunk 1]
```diff
@@ -8,6 +8,7 @@ package scaladsl
 import java.util.concurrent.atomic.AtomicLong
 
 import akka.actor.ExtendedActorSystem
+import akka.actor.typed.Props
 import akka.actor.typed.internal.adapter.{ActorRefAdapter, PropsAdapter}
```
Comment (ennru) [inline mqtt-streaming/src/main/scala/akka/stream/alpakka/mqtt/streaming/scaladsl/MqttSession.scala]: Instead, Alpakka should use the public API version of `PropsAdapter` and the `ActorRefAdapter` shouldn't be used, either.

File: mqtt-streaming/src/main/scala/akka/stream/alpakka/mqtt/streaming/scaladsl/MqttSession.scala [hunk 2]
```diff
@@ -496,7 +499,9 @@ final class ActorMqttServerSession(settings: MqttSessionSettings)(implicit mat:
                               producerPacketRouter,
                               publisherPacketRouter,
                               unpublisherPacketRouter,
-                              settings)
+                              settings),
+            Props.empty,
+            true
```
Comment (huntc) [inline mqtt-streaming/src/main/scala/akka/stream/alpakka/mqtt/streaming/scaladsl/MqttSession.scala]: Named param?

File: mqtt-streaming/src/main/scala/akka/stream/alpakka/mqtt/streaming/scaladsl/MqttSession.scala (hunk 1 above)
Comment (jcroig) [reply mqtt-streaming/src/main/scala/akka/stream/alpakka/mqtt/streaming/scaladsl/MqttSession.scala]: Done.

File: mqtt-streaming/src/main/scala/akka/stream/alpakka/mqtt/streaming/scaladsl/MqttSession.scala (hunk 2 above)
Comment (jcroig) [reply mqtt-streaming/src/main/scala/akka/stream/alpakka/mqtt/streaming/scaladsl/MqttSession.scala]: After last commit these lines changed so I don't thing this applies anymore, thanks.

**Comment by ennru** [comment]
We should have detected this earlier, our scheduled build is still on Akka 2.6.0...
Thank you for fixing it.

**Comment by jcroig** [comment]
Pleasure.

**Pull request merged**